ansible/roles/airflow/files/airflow-dags/analysis.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+#
+# Use the existing shovel_jump.sh script to start the analysis script
+#
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from datetime import datetime, timedelta
+
+dag = DAG(
+    dag_id="analysis_dag",
```

Comment:
Minor: cursory looking at other DAGs seem to suggest we don't use the `_dag` prefix there. So, I'd recommend to avoid using the same prefix also here.